my_list.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&number)
```

Comment:
- [ ] It would be better if we implement this functionality -> kindly implement `#each` method that yields all members of `@list` in succession.

Reply:
I really do not understand what you are asking us to do? Could you, please, clarify?

Reply:
Hi guys, kindly go ahead. I guess you implemented it in your own way and it works fine

Reply:
Hi @addod19. If we could use yield it could throw as an error.

Reply:
So, I assume, we could submit for the second review after correcting the first two issues and leave the third one as it is.